scripts/scaffold_tests.py

```
@@ -0,0 +1,225 @@
+    ]
+
+    # Import the primary module(s)
+    has_any = False
+    for fname in sorted(py_files):
+        if not py_files[fname]:
+            continue
+        has_any = True
+        # Derive import path: e.g. phases/14-.../01-.../code/main.py
+        # We can't easily import from here, but we can reference the path.
+        lines.append("")
+        lines.append(f"# To import the code under test, add the lesson")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)